models/ens/ens_set_name_latest.sql

```diff
@@ -0,0 +1,84 @@
+{{
+    config(
+         alias = alias('set_name_latest')
+        ,tags = ['dunesql']
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['address', 'name']
```

Comment:
The uinque_key here should match the columns listed in the uniqueness test.

Reply:
Thank you for your patience.